src/ontology/cl-edit.owl

```diff
@@ -32366,6 +32367,18 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37124720") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033091 "glycinergic-GABAergic amacrine cell")
 EquivalentClasses(obo:CL_4033091 ObjectIntersectionOf(obo:CL_0000561 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061537)))
 
+# Class: obo:CL_4033092 (CD57-positive enterocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37468586") obo:IAO_0000115 obo:CL_4033092 "An enterocyte that is part of a duodenal epithelium and expresses beta-1,3-glucuronyltransferase 1 (B3GAT1), also known as CD57 or HNK-1.")
+AnnotationAssertion(terms:contributor obo:CL_4033092 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(terms:contributor obo:CL_4033092 <https://orcid.org/0000-0001-7655-4833>)
+AnnotationAssertion(terms:contributor obo:CL_4033092 <https://orcid.org/0000-0001-9961-7673>)
+AnnotationAssertion(terms:date obo:CL_4033092 "2024-12-06T14:32:49Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37468586") oboInOwl:hasExactSynonym obo:CL_4033092 "CD57+ enterocyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37468586") rdfs:comment obo:CL_4033092 "While CD57 is traditionally associated with immune cells (e.g., NK cells), its expression on enterocytes marks unique epithelial subsets with specific functions or maturation states. These cells are speciallyenriched in areas of the duodenum within submucosal glands.")
```

Comment:
Everything looks good. There is only a typo in the comments section - should have a space here `speciallyenriched`